sleep_sort/sleep_sort.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+// SleepSort sorts an array using SleepSort algorithm
+func SleepSort(numbers []int) []int {
+
+	channel := make(chan int, len(numbers))
+
```

Comment:
Hey, @hloeffler. Thanks for the PR. Everything looks good. 
Just one thing, I see a lot of empty lines mixed with the code. Do you see any advantage to this? 
The tests are passing so it won't be right to enforce removing these lines but if you agree with the same, please go ahead and do so.
Thanks